Modules/TreasureHunt/templates/TreasureHunt/intro.latte

```diff
@@ -1,5 +1,19 @@
 {block content}
-Hi
 
+{foreach $flashes as $flash}
+    <div class="flash {$flash->type}">{$flash->message}</div>
+{/foreach}
+
+<h1>Vítejte ve hře o poklad.</h1>
+
+<p>Formulářem níže se zaregistrujte nebo přihlašte.</p>
+<p>Jako heslo vyberte libovolnou kombinaci symbolů. :)</p>
+
+
+{* introForm:  \Modules\TreasureHunt\Presenters\TreasureHuntPresenter.php *}
 {control introForm}
+
+<pre>~~Šablona: /Modules/TreasureHunt/templates/TreasureHunt/intro.latte~~</pre>
```

Comment:
Odeber odsud, prosím, výpis souboru šablony.

Uvítal bys kdyby se dalo na každém pohledu zobrazit, jaký soubor šablony je zrovna použit v podobě panelu v liště Tracy?